sympy/functions/elementary/complexes.py

```diff
@@ -38,6 +38,18 @@ class re(Function):
     0
     >>> re(im(x) + x*I + 2)
     2
+    >>> re(5 + I + 2)
+    7
+
+    Parameters
+    ==========
```

Comment:
We would generally add one spacing between paragraph dashes and next like
```suggestion
    ==========

```

Reply:
Done